packages/create-just/src/commands/initCommand.ts

```diff
@@ -107,12 +111,14 @@ Please make sure you have git installed and then issue the following:
 
   logger.info('All Set!');
 
-  showNextSteps(argv, stackName);
+  showNextSteps(argv, stackName, stackPath!);
 }
 
-function showNextSteps(argv: any, stackName: string) {
-  logger.info(
-    prettyPrintMarkdown(`
+function showNextSteps(argv: any, stackName: string, stackPath: string) {
+  const postInitPath = path.join(stackPath, 'PostInit.md');
+  const nextStepsMd = existsSync(postInitPath)
+    ? readFileSync(postInitPath, 'utf-8').toString()
+    : `
```

Comment:
we can possibly just put the content of this in a "defaultpostgen.md"

Reply:
I dont think the default can be in a md file, since its got a bunch of runtime strings in it.  Unless I'm misunderstanding you?

Reply:
let's support templates via handlebars later (separate change).